com.unity.netcode.gameobjects/Runtime/Messaging/RpcParams.cs

```diff
@@ -25,7 +25,7 @@ public struct ClientRpcSendParams
         /// Note: Even if you provide a value such as NativeArray, enumerating it will cause boxing.
         /// If you want to avoid boxing, use TargetClientIdsNativeArray
         /// </summary>
-        public IEnumerable<ulong> TargetClientIds;
+        public IReadOnlyList<ulong> TargetClientIds;
```

Comment:
If this becomes IReadOnlyList, then NetworkBehaviour.SendClientRpc() can be adjusted to not have to make a copy of it and just pass it directly to SendMessage() :)

Reply:
Yes you are right, simplified the code to use the SendMessage overload with IReadOnlyList.